easybuild/easyconfigs/v/ViennaRNA/ViennaRNA-2.4.10-foss-2018b.eb

```diff
@@ -0,0 +1,54 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'ViennaRNA'
+version = '2.4.10'
+
+homepage = 'http://www.tbi.univie.ac.at/RNA/'
+description = """The Vienna RNA Package consists of a C code library and several
+stand-alone programs for the prediction and comparison of RNA secondary structures."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.tbi.univie.ac.at/RNA/download/sourcecode/%(version_major)s_%(version_minor)s_x/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['87c1d8f25009925baa0dfae06d635ad39eefe930e372842b412b118a10ecc15c']
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('flex', '2.6.4'),
+    ('pkg-config', '0.29.2'),
+    ('Bison', '3.0.5'),
+    ('SWIG', '3.0.12', '-Python-%(pyver)s'),
+]
+
+dependencies = [
+    ('Python', '2.7.15'),
```

Comment:
@hajgato Since `Python` is a very common dependency, please add the customary `versionsuffix`.

For Perl that's much less of a problem, since we stick to a single version of `Perl` per toolchain generation anyway.